ColabTool/Shared/Post/index.jsx

```diff
@@ -42,10 +68,7 @@ const Post = ({
         <PostHeader
           draftDetails={draftDetails}
         />
-        <div className={style['post-content']}>
-          {renderRetweetPanel(retweetProfile)}
-          {children}
-        </div>
+        {renderRetweet(children, retweetProfile)}
```

Comment:
Hey @stevemdixon digging into the code it looks like this renders the post content and conditionally the retweet details (which feels like its part of the post content). A more descriptive name seems like `renderContent` to me